opentreemap/treemap/js/src/lib/inlineEditForm.js

```diff
@@ -19,6 +19,7 @@ exports.init = function(options) {
         form = options.form || section + ' form',
         $section = $(section),
         edit = options.edit || '.editBtn',
+        $edit = $(edit),
```

Comment:
A single instantiation of an inline edit form can persist as the fields it manages are reloaded via AJAX -- in particular, the details sidebar of the main map page. For that reason the inline edit form code doesn't hang on to $ variables (other than $section, which is the container) but rather makes them when needed. 

In the details sidebar case the "edit" button is outside of the partial retrieved by AJAX so the code will probably work fine. But I would personally stick with the existing style (use `$(edit)` instead of `$edit`), and possibly add a comment for future developers.